### PR TITLE
docs for deploy to OpenShift

### LINKED ISSUE #354: instructions for running appsody on OpenShift Red Hat CodeReady Containers
**Is your feature request related to a problem? Please describe.**
Instructions are needed for deploying on a local OpenShift environment. This can be done using Red Hat CodeReadyContainers.

### PR DESCRIPTION
**Pull request by tnixa**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description
Documentation for `appsody deploy` to local OpenShift environment

## Related Issues
fixes #354

### CODE REVIEW

File: content/docs/using-appsody/building-and-deploying.md [hunk 1]
```diff
@@ -224,6 +224,188 @@ This command completes the following actions:
...
+1. Find the `image-registry` address:
+    - Navigate to Workloads-> Pods
+    - Click on the main `image-registry` pod (Example: `image-registry-6984bcdf68-vpfjt`)
+    - Click on the `Environment` tab and scroll down to `REGISTRY_OPENSHIFT_SERVER_ADDR` and note the value (Example: `image-registry.openshift-image-registry.svc:5000`)
+    This value will be used for `--pull-url` for `appsody deploy`
+1. Find the `default-route` location:
+    - Navigate to Networking-> Routes
+    - Note the `Location` for the `default-route` (Example: `https://default-route-openshift-image-registry.apps-crc.testing`)
+    This value will be used for `--push-url` for `appsody deploy`
+1. Create the certificate (Red Ha Linux example):
```
Comment (helenmasters) [inline content/docs/using-appsody/building-and-deploying.md]: Red Ha -> Red Hat

File: content/docs/using-appsody/building-and-deploying.md [hunk 2]
```diff
@@ -224,6 +224,188 @@ This command completes the following actions:
...
+        Copy the `Server certificate` which starts with header: `-----BEGIN CERTIFICATE-----` and ends with header: `-----END CERTIFICATE-----` You must include the headers.
+    - Create the certificate directory: 
+        ```
+        sudo mkdir /etc/docker/certs.d/default-route-openshift-image-registry.apps-crc.testing
+        ```
+    - Create the certificate:
+        ```
+        vi /etc/docker/certs.d/default-route-openshift-image-registry.apps-crc.testing/ca.crt
+        ```
+    - Insert the copied certificate using `i` to get into insert mode and then paste the copied certificate and then press `esc` then `:wq` `<enter>` to save the file
```
Comment (helenmasters) [inline content/docs/using-appsody/building-and-deploying.md]: esc -> Esc

**Comment by tnixa** [comment]
> We cannot merge this into the deployment doc, as the current doc is already in need of splitting up.
> I'd also question whether this is more of a tutorial?
> I'll raise an issue now to work on splitting out deployment referencing this PR and issue.

I wasn't sure where this fit either. I think we have a "Tekton Example" so maybe this is more of an "Example"? But it is a deployment environment so it does seem to fit in with the other deployment stuff.

**Comment by skoh7645** [comment]
Closing this for now and this PR contains a lot of non-Appsody specific documentation. We will include the relevant parts of this PR in the fix for https://github.com/appsody/website/issues/359

**Comment by skoh7645** [comment]
Reopening in hopes to verify and simplify this PR

**Comment by neeraj-laad** [comment]
Closing as no longer relevant. Other changes have been made to build/deploy/ and reference sections that cover this.

**Pull request closed without merging**